security/sequoia-sq/Portfile

```diff
@@ -0,0 +1,563 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           gitlab 1.0
+PortGroup           cargo 1.0
+
+gitlab.setup        sequoia-pgp sequoia-sq 0.38.0 v
+checksums           ${distname}${extract.suffix} \
+                    rmd160  6c7e9654c10966a5a100f4a09911d1492c1c82e0 \
+                    sha256  d30f45e66eb227ede3df237e370016023b709966dfe05d33a062582dc308411b \
+                    size    498719 \
+
+description         \
+    Sequoia-PGP is an implementation of OpenPGP in Rust. This port provides \
+    the sq command line application, which provides a way to use OpenPGP \
+    conveniently from the command line.
+long_description    ${description}
+license             LGPL-2.0+
+homepage            https://sequoia-pgp.org/
+
+categories          security mail
+maintainers         {cal @neverpanic} openmaintainer
+
+depends_build-append \
+                    port:capnproto \
+                    port:pkgconfig
+
+depends_lib-append  \
+                    port:libiconv \
+                    port:nettle \
+                    port:openssl3 \
```

Comment:
Should this perhaps be the `openssl` portgroup instead?

Reply:
This seems to build fine as-is, so I'm not sure I actually need all the 266 lines of Tcl complexity introduced by the openssl PortGroup?